checker/src/abstract_value.rs

```diff
@@ -226,6 +226,15 @@ impl AbstractValue {
         }
     }
 
+    /// Returns an abstract value whose corresponding set of concrete values include all of the
+    /// values resulting from applying "as target_type" to each element of the concrete values of self.
```

Comment:
Can you use the word 'values' a couple more times here, for good measure? :)

Reply:
No, can't bring myself to do it. ;-)